build/package/Microsoft.DotNet.Cli.Nupkg.targets

```diff
@@ -11,46 +11,111 @@
     </PropertyGroup>
     
     <ItemGroup>
-      <ProjectsToPack Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.Cli.Utils" >
+      <MicrosoftDotNetCliUtilsInputs Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.Cli.Utils/**/*.cs;
+                                              $(ProjectsSrcDirectory)/Microsoft.DotNet.Cli.Utils/**/project.json;
```

Comment:
You don't really need **/project.json, since there is only a project.json at the root